packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -714,6 +713,11 @@ export class Relayer {
     };
   }
 
+  private async getCodeDeployedAtTimestamp(address: string, timestamp: number): Promise<string> {
```

Comment:
I really dont like that this has to call `getBlockForTimestamp`. I feel storing a list of constants in the file is better.

Prehaps you should call this _once_ on the initialization of the pool within the L1Client?

Reply:
Hm yeah it's a trade off of more hard coding and required environment variables vs runtime. 

I think what I'll do is open another PR where I just make the changes to the BridgePool and then leave this open as a potential bot "optimization"